app/views/businesses/new.html.erb

```diff
@@ -2,5 +2,6 @@
   <%= f.text_field :name, placeholder: "Name" %>
   <%= f.text_field :about, placeholder: "About" %>
   <%= f.file_field :picture %>
+  <%= f.hidden_field :user_id, value: current_user.id %>
```

Comment:
This is potentially insecure as it lets the user know something about the structure of the database and if the authenticity token isn't working could result in a csrf attack